test/unit/parse-comment.test.js

```diff
@@ -62,6 +62,17 @@ describe('parseComment', () => {
       })
   })
 
+  test('Basic intent to add - username with multiple dashes', () => {
+    expect(
+        parseComment(`@${testBotName} please add rishi-raj-jain for doc`),
+    ).toEqual({
+        action: 'add',
+        contributors: {
+            "rishi-raj-jain": ['doc'],
+        },
+    })
+})  
```

Comment:
The test is not failing. Looks like the existing code is working, even without your change?

Reply:
here is a test, seems like the username is parsed okay?

https://github.com/gr2m/sandbox/issues/252#issuecomment-1539284970

Reply:
Hmm, that's where I traced it down to parsing the username. It shows a valid username not being parsed here https://github.com/analogjs/analog/pull/362#issuecomment-1529114039

Reply:
hm okay I'll do some more testing

Reply:
okay I was able to reproduce the problem: https://github.com/gr2m/sandbox/issues/252#issuecomment-1546400131.

As I said the test is not failing the implementation of your fix though. Could you please have a look? I'll meanwhile deploy your changes to staging and test that

Reply:
confirmed your change is working: https://github.com/gr2m/sandbox/issues/252#issuecomment-1546408394. So all it's left is to make your test fail before the fix.

Reply:
Sure, thanks